### DISCUSSION TITLE
500 Internal Server error on button click

### DISCUSSION THREAD

**Discussion by arunprad** [Budibase/budibase#5072] [Help]
I'm facing the same issue as https://github.com/Budibase/budibase/issues/4938

I see this was fixed as well. Is there any info on when this will be pushed to the docker container?

Or please point me towards docs on how to build and run budibase from source code.

Thanks!

**Comment by mjashanks** [comment]
This will be released next week!

**Comment by hawkeye426** [comment]
Has this been fixed. We are still experiencing as similar issue when attempting to load any RESTFUL data source.
